templates/ci/.net-desktop.yml

```diff
@@ -0,0 +1,51 @@
+# .NET Desktop
+# Build and run tests for .NET Desktop or Windows classic desktop solutions.
+# https://aka.ms/yaml
+
+queue: 'Hosted VS2017'
+
+variables:
+  buildConfiguration: 'Release'
+  buildPlatform: 'Any CPU'
+  solution: '**/*.sln'
```

Comment:
quotes would be needed here since leading * means something special in yaml

Reply:
Got it